scalajvm/app/helpers/Cors.scala

```diff
@@ -0,0 +1,26 @@
+package helpers
```

Comment:
"Helpers" and "Utilities" are instances of coincidental cohesion which is the worst possible cohesion according to Larry Constantine https://en.wikipedia.org/wiki/Cohesion_(computer_science)#Types_of_cohesion

I will create an issue to eliminate this package, but for I now I strongly recommend to create `cors` package and move everything related to CORS there.